starter-core/src/main/java/io/micronaut/starter/application/generator/GeneratorContext.java

```diff
@@ -186,10 +186,13 @@ public void applyFeatures() {
     }
 
     public boolean isFeaturePresent(Class<? extends Feature> feature) {
-        Objects.requireNonNull(feature, "The feature class cannot be null");
```

Comment:
Why was this removed?

Reply:
`features.isFeaturePresent` has the same check, so it was redundant